### PR TITLE
Feature/gloria iwd form v1.0

### PR DESCRIPTION
**Pull request by GDesigns49**
A form designed with HTML, CSS and JavaScript to accept and display list of inputted names,

### CODE REVIEW

File: FirstCodingTask/index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Awesome Women Form</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <Form align="center">
```
Comment (gwa-admin) [inline FirstCodingTask/index.html:11]: This should be lower-case, so it should be `<Form>` not `<form>`

**Review by gwa-admin — COMMENTED** [review]
Decent attempt @GDesigns49, I've added some comments to your PR but here's a summary.

- The logic doesn't work as expected. When I add a name and click on the **ADD** button, it doesn't add to the list. Do well to verify this.

File: Myfirstcommit.go
[the remark this answers is not in the document]
Comment (GDesigns49) [reply Myfirstcommit.go:1]: ok sir

File: FirstCodingTask/index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            <label for="task"><h3>List of Awesome Women I know</h3></label>
+                <br>
+            <div class="task">     
+            </div>
+        
+        </div>
+            
+    </Form>
+
+
```
[the remark this answers is not in the document]
Comment (GDesigns49) [reply FirstCodingTask/index.html]: yes sir

File: FirstCodingTask/index.html (hunk 1 above)
Comment (GDesigns49) [reply FirstCodingTask/index.html]: Noted sir

**Comment by GDesigns49** [comment]
Sir,

With reference to the above subject matter, I wish to state that all
corrections and comments have been taken and noted down and will be treated
accordingly.

Please sir, I wish to request your permission to effect all corrections
and also respond to your comments on my PR three (3) to four (4) days after
the forth coming Governorship election. The reason for my  request is my
involvement in the uploading of the election results into a particular
server. Hence, I have been utterly busy since yesterday being (15th of
March, 2023) and will only be less busy after all results have been
uploaded into the server in question.

Hope my request get granted.

Thank you for your time.
Warm regards.

**Comment by gwa-admin** [comment]
Hello @GDesigns49,  closing this PR to cleanup the repository for the project, just FYI.

**Pull request closed without merging**